src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $secondary-accent;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
+
+.product-card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review-count {
+    color: #060b35;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent;
```

Comment:
The variable `$main-accent` is used here again, but it's not defined within this file. Ensure that it is defined in a variables file that is imported, or define it here if it's missing.